### DISCUSSION TITLE
Missing "fracVegCover.map" file within filepath: $(FILE_PATHS:PathMaps)/landcover/

### DISCUSSION THREAD

**Discussion by Aichumei** [iiasa/CWatM#130] [Q&A]
hello,
i am learning the setting files of model to add some new files of land cover types. 
I tried to search "fracVegCover.map" in [settings_CWatM_template_30min.ini] in the GitHub repository, however, I could not find any. Is it the same with or from fractionLandcover.nc within filepath: $(FILE_PATHS:PathMaps)/landsurface/ ?
Thank you very mouch!

**Comment by mikhailsmilovic** [comment]
Guten Tag, great question about updating the land cover

## Option 1: Dynamic
The correct dynamic land cover is pointed to in the settings file at:
fractionLandcover = $(PathLandcover)/fractionLandcover.nc
This file is in CWatM-Earth-30min\landsurface
One could copy/update this map collection time series, holding multiple years and all land covers.

## Option 2: Static
Another option is to input static maps, TIF or NetCDF, for each land cover.
For this, in the [OPTIONS] section, include:
**staticLandCoverMaps = True**

Then below in [LANDCOVER], include:
**forest_fracVegCover  = _points to forest map_**
**irrPaddy_fracVegCover  = _points to irrigated paddy map_**
**irrNonPaddy_fracVegCover  = _points to irrigated (non-paddy) map_**
**sealed_fracVegCover = _points to sealed map_**
**water_fracVegCover  =  _points to water map_**
